.github/workflows/python_format.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
```

Comment:
Will this run on every push even if not part of a PR?

Reply:
Good point. Updated to trigger only push to master, or on PRs to any branch (still only when python files change).

Reply:
Demonstrated desired behavior in #344.